validator/src/main/resources/expected-data-template/expectedAWSSDKTrace.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "/aws-sdk-call",
+  "fault": false,
+  "error": false,
+  "throttle": false,
+  "http": {
+    "request": {
+      "url": "http://127.0.0.1:8080/aws-sdk-call",
```

Comment:
why the ip is 127.0.0.1? i imagine the ip should be the ip of the service?

Reply:
good point. maybe we can provide env variable to set this value? because url will be depending on which platform we deploy application right? so we won't be having url value on hand.